client/jetpack-cloud/sections/manage/pricing/index.web.js

```diff
@@ -10,4 +11,12 @@ export default function () {
 		makeLayout,
 		clientRender
 	);
+
+	page(
+		`/:lang` + jetpackManagePricingPath(),
```

Comment:
Can this logic be placed within `jetpackManagePricingPath()`?

Reply:
Right now it's just a function that returns a string; I can probably move the logic into said function to return the string variant based on some param, though, if that's what you mean? (e.g. `jetpackManagePricingPath( true )`).

Reply:
On second thought, I've just moved the path directly into the function, which is more readable: 7a81e5300a

It didn't appear to be used anywhere else anyway.

Reply:
> which is more readable

100%